src/styles/blocks/card.scss

```diff
@@ -0,0 +1,84 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  background-color: $mainBackgroundColor;
+  width: 100%;
+  max-width: 200px;
+  padding: 32px 16px 16px;
+  border: 1px solid $mainBorderColor;
+  border-radius: 5px;
+
+  &__img {
+    height: 134px;
+    margin: 0 auto 40px;
+  }
+
+  &__title {
+    color: $mainFontColor;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    margin-bottom: 4px;
+  }
+
+  &__product-code {
+    color: $secondaryFontColor;
+    font-size: 10px;
+    line-height: 14px;
+    margin-bottom: 16px;
+  }
+
+  &__review {
+    display: flex;
+    justify-content: space-between;
+    align-items: flex-end;
+
+    &-reviews {
+      color: $mainFontColor;
+      font-size: 10px;
+      line-height: 14px;
+    }
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    align-items: flex-end;
+
+    &-title {
+      color: $secondaryFontColor;
+      font-size: 12px;
+      line-height: 18px;
+      margin-top: 24px;
+    }
+
+    &-count {
+      color: $mainFontColor;
+      font-weight: 700;
+    }
+  }
+
+  &__button {
+    display: flex;
+    color: $buttonTextColor;
+    background-color: $buttonBackgroundColor;
+    font-size: 14px;
```

Comment:
Let's try to organize your CSS
I recommend you take a look at these styles code and use it in the feature
[9elements](https://9elements.com/css-rule-order) or [htmlAcademy](https://codeguide.academy/html-css.html#css-order)